editoast/src/converters/utils.rs

```diff
@@ -511,4 +515,32 @@ mod tests {
         assert_eq!(1., reference_coord(NodeId(0), &edge).lon);
         assert_eq!(0., reference_coord(NodeId(2), &edge).lon);
     }
+
+    #[rstest]
+    #[case("15000", "15000V")]
+    #[case("15000V", "15000V")]
```

Comment:
We may add a test case on "weird" value:
```rust
    #[case("weird value", "weird value")]  // unspecified case: electrification could also be ignored
```

This would also be proposed in rolling-stock editor, but we can merge as-is IMO.